packages/svelteui-core/src/components/Chip/Chip.styles.ts

```diff
@@ -0,0 +1,189 @@
+};
+
+const padding = {
+  xs: 16,
+  sm: 20,
+  md: 24,
+  lg: 28,
+  xl: 32,
+};
+
+const checkedPadding = {
+  xs: 7.5,
+  sm: 10,
+  md: 11.5,
+  lg: 13,
+  xl: 15,
+};
+
+export default createStyles(
+	(theme, { color, radius, size, transitionDuration, variant }: ChipStyleParams, getRef) => {
+		return {
+      root: {},
+
+      label: {
+        ref: getRef('label'),
+        boxSizing: 'border-box',
+        color: theme.colors.black.value,
+        darkMode: {
+          color: theme.fn.themeColor('dark', 0)
+        },
+        display: 'inline-block',
+        alignItems: 'center',
+        userSelect: 'none',
+        border: '1px solid transparent',
+        borderRadius: theme.fn.radius(radius),
+        height: theme.fn.size({ size, sizes }),
+        fontSize: theme.fn.size({ size, sizes: theme.fontSizes }),
+        lineHeight: `${theme.fn.size({ size, sizes }) - 2}px`,
+        paddingLeft: theme.fn.size({ size, sizes: padding }),
+        paddingRight: theme.fn.size({ size, sizes: padding }),
+        cursor: 'pointer',
+        whiteSpace: 'nowrap',
+        transition: `border-color ${transitionDuration}ms ease, background-color ${transitionDuration}ms ease`,
+        WebkitTapHighlightColor: 'transparent',
+
+        '&.outline': {
+          darkMode: {
+            border: `1px solid ${theme.fn.themeColor('dark', 4)}`,
+            backgroundColor: theme.fn.themeColor('dark', 6),
+          },
+          border: `1px solid ${theme.fn.themeColor('gray', 4)}`,
+          backgroundColor: theme.colors.white.value,
+        },
+
+        '&.filled': {
+          darkMode: {
+            backgroundColor: theme.fn.themeColor('dark', 4),
+          },
+          backgroundColor: theme.fn.themeColor('gray', 1)
+        },
+        '&:hover': {
+          backgroundColor: theme.fn.themeColor('gray', 2),
+
+        },
+        '&.disabled': {
+          backgroundColor: `${ theme.fn.themeColor('gray', 1) } !important`,
+          borderColor: `${theme.fn.themeColor('gray', 1)} !important`,
+          color: theme.fn.themeColor('gray', 5),
+          darkMode: {
+            backgroundColor: `${theme.fn.themeColor('dark', 5)} !important`,
+            borderColor: `${theme.fn.themeColor('dark', 5)} !important`,
+            color: theme.fn.themeColor('dark', 3),
+          },
+          cursor: 'not-allowed',
+
+          // this isn't working ?? the checkmark remains coloured when disabled
+          [`& .${getRef('iconWrapper')}`]: {
+            color: theme.fn.themeColor('dark', 5),
+            darkMode: {
+              color: theme.fn.themeColor('dark', 3)
+            }
+          },
+        },
+
+        '&.checked': {
+          paddingLeft: theme.fn.size({ size, sizes: checkedPadding }),
+          paddingRight: theme.fn.size({ size, sizes: checkedPadding }),
+
+          '&.outline': {
+            border: `1px solid ${theme.fn.themeColor(color, 5)}`,
+          },
+
+          '&.filled': {
+            '&, &:hover': {
```

Comment:
Small typo here

Reply:
Can't see the typo, I'm quite new to stitches. I've moved the background color to be in both `&.filled` and `&:hover`.

Reply:
The part below with `&,`